app/src/main/res/layout-land/profile_edit_activity.xml

```diff
@@ -0,0 +1,240 @@
+          android:textAllCaps="false"
+          android:textColor="@color/red"
+          android:textSize="16sp" />
+
+        <View
+          android:layout_width="match_parent"
+          android:layout_height="1dp"
+          android:background="@color/grey" />
+      </LinearLayout>
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/profile_edit_main"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/white"
+        android:clipToPadding="false"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <LinearLayout
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="24dp"
+          android:layout_marginEnd="8dp"
+          android:orientation="vertical"
+          app:layout_constraintBottom_toBottomOf="@id/profile_edit_image"
+          app:layout_constraintEnd_toEndOf="parent"
+          app:layout_constraintHorizontal_bias="0.0"
+          app:layout_constraintStart_toEndOf="@id/profile_edit_image"
+          app:layout_constraintTop_toTopOf="@id/profile_edit_image">
+
+          <TextView
+            android:id="@+id/profile_edit_name"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:fontFamily="sans-serif-medium"
+            android:maxLines="1"
+            android:text="@{viewModel.profile.name}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="20sp"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+          <TextView
+            android:id="@+id/profile_edit_created"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:fontFamily="sans-serif"
+            android:maxLines="1"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="16sp"
+            profile:created="@{viewModel.profile.dateCreatedTimestampMs}" />
+        </LinearLayout>
+
+        <de.hdodenhof.circleimageview.CircleImageView
+          android:id="@+id/profile_edit_image"
+          android:layout_width="90dp"
+          android:layout_height="90dp"
+          android:layout_marginStart="72dp"
+          android:layout_marginTop="32dp"
+          android:layout_marginBottom="36dp"
+          app:layout_constraintBottom_toBottomOf="parent"
+          app:layout_constraintStart_toStartOf="parent"
+          app:layout_constraintTop_toTopOf="parent"
+          profile:src="@{viewModel.profile.avatar}" />
+      </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.constraintlayout.widget.ConstraintLayout>
+  </ScrollView>
+
+  <data>
```

Comment:
Please keep `data` above `scrollView`.

Reply:
@rt4914 sorry it was made automatically by android studio. I changed them as you mentioned.
Thanks...